src/CommonLib/Exceptions/SharpHoundCommonException.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace SharpHoundCommonLib.Exceptions
+{
+    public class SharpHoundCommonException : Exception
```

Comment:
Can we rename this to something thats more specific to our problem set, like maybe LDAPQueryException or something. SharpHoundCommonException is a bit too generic

Reply:
Absolutely. I think LDAPQueryException probably makes the most sense here honestly, so I'll refactor that quick.